temporalcli/commands_test.go

```diff
@@ -61,6 +61,10 @@ func (h *CommandHarness) ContainsOnSameLine(text string, pieces ...string) {
 	h.NoError(AssertContainsOnSameLine(text, pieces...))
 }
 
+func (h *CommandHarness) AnyLineMatchesRegexp(text string, pattern string) {
```

Comment:
Can you just use `h.Regexp`?

Reply:
I... did not realize that was a thing, oops. Yes, will fix.